File: src/main/java/discord/commands/container/GP.java
package discord.commands.container;


import discord.commands.DiscordCommand;
import discord.entity.DiscordUser;
import discord.utils.Utils;
import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.events.message.MessageReceivedEvent;

public class GP extends DiscordCommand {

    @Override
    public void execute(EmbedBuilder builder, StringBuilder text, MessageReceivedEvent bot, DiscordUser user, String... cmd) {
        bot.getChannel().sendMessage("<:moneybag:718197721495765214>You have " + Utils.formatNumber(user.getGp()) + " GP!").queue();
    }

    @Override
    public String[] getCommand() {
        return new String[] { "-gp", "-coins", "-gold", "-goldpoints" };
    }

    @Override
    public String getDescription() {
        return "Shows the amount of GP you have in your discord account.";
    }

    @Override
    public String getArguments() {
        return null;
    }

}

File: src/main/java/discord/commands/container/kotlin/GP.kt
package discord.commands.container.kotlin

import discord.commands.DiscordCommand
import discord.entity.DiscordUser
import discord.utils.Utils
import net.dv8tion.jda.api.EmbedBuilder
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class GP : DiscordCommand() {

    override fun execute(builder: EmbedBuilder, text: StringBuilder, bot: MessageReceivedEvent, user: DiscordUser, vararg cmd: String) {
        bot.channel.sendMessage("<:moneybag:718197721495765214> You have ${Utils.formatNumber(user.gp)} GP!").queue()
    }

    override fun getCommand(): Array<String> {
        return arrayOf("-gp", "-coins", "-gold", "-goldpoints")
    }

    override fun getDescription(): String {
        return "Shows the amount of GP you have in your discord account."
    }

    override fun getArguments(): String? {
        return null
    }

}